inject.js

```diff
@@ -140,6 +140,7 @@
           var speed = this.getSpeed();
           this.speedIndicator.textContent = speed;
           tc.settings.speeds[this.video.src] = speed;
+          tc.settings.speed = speed;
```

Comment:
What's your theory or thinking behind this one? I similarly haven't been able to reproduce this bug..

Reply:
I can reproduce the bug specifically mentioned in #455, but not the potentially related #457 or #459. It is caused by line 131 in the event listener for 'play.' I added that preamble to initialize the speed in case it had not yet been initialized. But in changing to the Map of speeds for each source I removed all updating of tc.settings.speed. This updates tc.settings.speed always to the last speed.

I feel like we should refactor settings.speed to settings.lastSpeed since that is now the remaining meaning of it. I can go ahead and do that if you agree.

Reply:
To confirm, did you publish the current master (through 5fec357) the other day? Are these bugs in the latest code or something else?

Reply:
Yes, latest release includes the audio PR.

Let's refactor this, I don't think it makes sense to maintain both — only adds to confusion.